docs/detections/rules-prebuilt.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[rules-prebuilt]]
+== Install Elastic prebuilt rules
+
+:frontmatter-description: Start detections quickly with prebuilt rules designed and updated by Elastic.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [how-to]
+:frontmatter-tags-user-goals: [manage]
+
+Follow these guidelines to start using the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, keep them updated, and make sure they have the data needed to run successfully. 
+
+* <<load-prebuilt-rules>>
+* <<prebuilt-rule-tags>>
+* <<select-all-prebuilt-rules>>
+* <<update-prebuilt-rules>>
+* <<download-prebuilt-rules>>
+* <<rule-prerequisites>>
+
+[NOTE]
+====
+* Apart from the Elastic Endpoint rule, prebuilt rules do not start running by default. You must first install the rules, then enable them.
+
+* You can't modify most settings on Elastic prebuilt rules. You can only edit <<rule-schedule, rule actions>> and <<add-exceptions, add exceptions>>. If you want to modify other settings on a prebuilt rule, you must first duplicate it, then make your changes to the duplicated rule. 
+====
+
+[float]
+[[load-prebuilt-rules]]
+=== Install and enable Elastic prebuilt rules
+
+. Go to *Manage* -> *Rules*. The badge next to *Add Elastic rules* shows the number of uninstalled prebuilt rules available. 
++
+[role="screenshot"]
+image::images/prebuilt-rules-add-badge.png[The Add Elastic Rules page]
+
+. Click *Add Elastic rules*, then do one of the following:
+* Install all available rules: Click *Install all*.
+* Install a single rule: Click *Install rule* for that rule.
+* Install multiple rules: Select the rules and click *Install _x_ selected rule(s)*.
++
+TIP: Use the search bar and *Tags* filter to find the rules you want to install. For example, filter by `OS: Windows` if your environment only includes Windows endpoints. For more on tag categories, refer to <<prebuilt-rule-tags>>.
++
+[role="screenshot"]
+image::images/prebuilt-rules-add.png[The Add Elastic Rules page]
+
+. Go back to the *Rules* page, then search or filter for any rules you want to start running, and do the following:
+
+* Enable a single rule: Turn on rule's the *Enabled* switch.
+* Enable multiple rules: Select the rules, then click *Bulk actions* -> *Enable*.
+
+Once you enable a rule, it starts running on its configured schedule. To confirm that it's running successfully, check its *Last response* status in the rules table, or open the rule's details page and check the <<rule-execution-logs, *Execution results*>> tab.
+
+[float]
+[[prebuilt-rule-tags]]
+=== Prebuilt rule tags
+
+Each prebuilt rule includes several tags identifying the rule's purpose, detection method, associated resources, and other information to help categorize your rules. These tags are category-value pairs, such as `OS: Windows` for a rule designed for Windows endpoints. Categories include:
```

Comment:
It could just be personal preference, but I think `for example` works slightly better than `such as` here. 

```suggestion
Each prebuilt rule includes several tags identifying the rule's purpose, detection method, associated resources, and other information to help categorize your rules. These tags are category-value pairs, for example, the `OS: Windows` tag indicates which rules are designed for Windows endpoints. Categories include:
```

Reply:
Slight modification:
```suggestion
Each prebuilt rule includes several tags identifying the rule's purpose, detection method, associated resources, and other information to help categorize your rules. These tags are category-value pairs; for example, `OS: Windows` indicates rules designed for Windows endpoints. Categories include:
```